android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -815,6 +967,11 @@ public int getInputType()
 		disableChangeEvent = false;
 	}
 
+	private boolean isSignChar(char value)
+	{
+		return value == '-' || value == '+';
```

Comment:
We also need to check for unicode minus signs `\u2212` and `\u2013 `.
This is based on what Google has coded here...
https://github.com/aosp-mirror/platform_frameworks_base/blob/master/core/java/android/text/method/DigitsKeyListener.java